### PR TITLE
[wip] Travis testing

### PR DESCRIPTION
**Pull request by matthewbauer**
Testing travis functionality

### CODE REVIEW

**Comment by grahamc** [comment]
Interesting, what are you going for?

**Comment by matthewbauer** [comment]
Well I've been trying to remove some of the extraneous stuff to give us more space for building. You can get about 5GB by just removing some stuff from apt-get, but I'm not really sure if it's enough to be successful.

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -6278,6 +6278,8 @@ in
 
   docutils = pythonPackages.docutils;
 
+  doctl = callPackage ../development/tools/doctl { };
```
Comment (domenkozar) [inline pkgs/top-level/all-packages.nix]: this doesn't belong here right? :)

Comment (matthewbauer) [inline pkgs/top-level/all-packages.nix]: Well I needed some failing commits to verify this fixes it.

Comment (domenkozar) [inline pkgs/top-level/all-packages.nix]: Ah ok :)

**Comment by domenkozar** [comment]
@matthewbauer no luck with this?

**Comment by matthewbauer** [comment]
Most of our problems with "no space left on device" are coming from /run/user/ which is a tmpfs and not from the ext4 partition. So removing all of the apt-get stuff is freeing up hard disk space but not memory. Perhaps we could just build everything straight on the ext4 partition instead, but I suspect that would slow everything down quite a bit.

For example: https://travis-ci.org/NixOS/nixpkgs/jobs/145846933

**Comment by domenkozar** [comment]
@matthewbauer take something that compiles longer and benchmark the difference if `/run/user` is not on tmpfs

**Comment by domenkozar** [comment]
But really, it's better if it's a bit slower than failing with space errors.

**Pull request closed without merging**